code/modules/maps/lateload/generic/unique_lateload.dm

```diff
@@ -0,0 +1,107 @@
+
+////////////////////////////////////////////////
+///////   Unique Map Templates   ///////////////
+////////////////////////////////////////////////
+
+/*
+This is for maps that are not overmap accessible but are also not gateway maps. There shouldn't be much of these right now
+unless a large shift in design philosophy occurs
+
+For overmap things that are planets and accessible from the overmap please put them
+in planets_lateload.dm
+
+For overmap things that are in space but *are not* planets put them in space_lateload.dm
+
+For gateway missions please put them in gateway_lateload.dm
+
+*/
+
+
```

Comment:
I'm tired of this. Put details of `map_template` only on the defile of `map_template` put the details of `unique_lateload` only on `unique_lateload` and etc.

Reply:
Done <3